docs/source/cc_service.md

```diff
@@ -0,0 +1,276 @@
+
+The rest of this topic describes how to configure chaincode as an external service:
+
+* [Packaging chaincode](#packaging-chaincode)
+* [Configuring a peer to process external chaincode](#configuring-a-peer-to-process-external-chaincode)
+* [External builder and launcher sample scripts](#external-builder-and-launcher-sample-scripts)
+* [Writing chaincode to run as an external service](#writing-chaincode-to-run-as-an-external-service)
+* [Deploying the chaincode](#deploying-the-chaincode)
+* [Running the chaincode as an external service](#running-the-chaincode-as-an-external-service)
+
+## Packaging chaincode
+
+With the Fabric v2.0 chaincode lifecycle, chaincode is [packaged](./cc_launcher.html#chaincode-packages) and installed in a `.tar.gz` format. The following `myccpackage.tgz` archive  demonstrates the required structure:
+
+```sh
+$ tar xvfz myccpackage.tgz
+code.tar.gz
+metadata.json
+```
+
+### code.tar.gz archive requirements
+
+The `code.tar.gz` archive must include connection information for the chaincode endpoint. This information is packaged into `connection.json` in the `/bin/release` step (see below). In this example we package `connection.json` directly in `code.tar.gz` so the `release` step can just copy it over.
+
+* **address** - chaincode server endpoint accessible from peer. Must be specified in “<host>:<port>” format.
+* **dial_timeout** - interval to wait for connection to complete. Specified as a string qualified with units (examples : "10s", "500ms", "1m"). Default is “3s” if not specified.
+* **tls_required** - true or false. If false "client_auth_required", "key_path", "cert_path", "root_cert_path" are not required. Default is “true”.
+* **client_auth_required** - if true then you need to specify "key_path" and "cert_path" for client authentication. Default is false. Ignored if tls_required is false.
+* **key_path** - path to the key file. This path is relative to the “release directory” (see “release” below). Required if client_auth_required is true. Ignored if tls_required is false.
+* **cert_path**  - path to the certificate file. This path is relative to the “release directory” (see “release” below). Required if client_auth_required is true. Ignored if tls_required is false.
+* **root_cert_path**  - path to the root cert for authenticating the server. Required when tls_required is set to true.
+
+For example:
+```json
+{
+  "address": "your.chaincode.host.com:9999",
+  "dial_timeout": "10s",
+  "tls_required": true,
+  "client_auth_required": "true",
+  "key_path": "path/rooted/in/release/directory/key.pem",
+  "cert_path": "path/rooted/in/release/directory/cert.pem",
+  "root_cert_path": "path/rooted/in/release/directory/rootcert.pem"
```

Comment:
There's a JIRA opened to change these to be values within the document instead of spreading them around the tree. That changes the key and the semantics and keep the definitions consistent between chaincode clients (what we have today) and chaincode servers (what this is talking about).

I really don't think it will make beta so this is mostly informational...